Studentmediene/kapina-backend
# Feat/kap 226 image credit

This PR introduces the following changes:
- Images added in CKEditor (The editor used for generating articles) now has a field for image creditting  when adding an image. Authors are required to fill out something in this field. By default it has the value "Foto: " to indicate the format that image credits usually take. 
- The image field in the Post model is now accompanied by another field called "Bildekredittering". This field is required to create an article. It includes a help text to indicate that the credits should be in the format "Foto: <navn>".
- GraphQL now has an `imageCredits` field on the `Post` model. This must be used in the frontend for populating the figcaption of the header image in an article.

To test this PR:
- Start Django locally
- Create an article, check that the image credit field is required when saving a model.  
- Insert an image in the CKEditor. Check that the image credit below the image is created.
